### DISCUSSION TITLE
Image sizes / recommendations

### DISCUSSION THREAD

**Discussion by 4kelly** [jaredkrinke/md2blog#11] [Q&A]
Im not very fluent in css, but the content width seems to max out at 640px, so it's best to resize images <= 640px? Are there any other general recommendations here?

Thanks!

**Comment by jaredkrinke** [comment]
Currently, there is no recommendation because I haven't thought it through yet. Ultimately, I'd like to just be able to throw any old image at md2blog and have it do something reasonable (e.g. display the image unmodified if it fits, otherwise automatically resize it and perhaps add a link to the full size image).

**Reply by jaredkrinke** [reply]
Opened #12 to track this. Thanks!

**Comment by jaredkrinke** [comment]
Forgot to update this thread, but here are recommendations:

https://jaredkrinke.github.io/md2blog/posts/faq/images.html
